openwisp_ipam/api/views.py

```diff
@@ -56,6 +56,11 @@ def get_csv_organization(self):
         return org
 
 
+class SerializerContextMixin:
+    def get_serializer_context(self):
+        return {'request': self.request}
```

Comment:
is this really needed? Doesn't this serializer context already get passed?

Reply:
Don't know why but the last time I tried, it wasn't passing the context. Tried again today and it did!